apps/dashboard/app/views/layouts/nav/_featured_apps.html.erb

```diff
@@ -1,4 +1,4 @@
-<% group = @featured_group %>
+<% group = @featured_group || OodAppGroup.new %>
```

Comment:
This seems like a code smell. It seems like if you're rendering this partial you should have a valid `@featured_group`.

I'll have to look further into how you may get into this state, but I'm guessing instead of catching it in the view here we should catch it somewhere else.